manifests/a/Amazon/AWSCLI/2.3.3.0/Amazon.AWSCLI.installer.yaml

```diff
@@ -0,0 +1,22 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+PackageIdentifier: Amazon.AWSCLI
+PackageVersion: 2.3.3.0
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: msi
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Commands:
+- aws
+ProductCode: "{AD383A13-C583-4717-8199-B6081EA9A817}"
```

Comment:
```suggestion
ProductCode: "{751D624B-C16E-4EBE-A085-74B49F683DA3}"
```

Reply:
Why was the ProductCode outside the installer entry? That should be a violation of intergalactic law.